src/routes/jira/jira-get-connections-backfillStatus.ts

```diff
@@ -0,0 +1,150 @@
+import { NextFunction, Request, Response } from "express";
+import { groupBy } from "lodash";
+import { RepoSyncState } from "~/src/models/reposyncstate";
+import {
+	TaskStatus,
+	Subscription,
+	SyncStatus
+} from "~/src/models/subscription";
+import { mapSyncStatus, ConnectionSyncStatus } from "./jira-get";
+
+type SubscriptionBackfillState = {
+	totalRepos?: number;
+	isSyncComplete: boolean;
+	syncedRepos?: number;
+	backfillSince?: string | null;
```

Comment:
curious what's the difference between undefined is null here? Could we plz either leave one of those or have clarifying explaining why?

Reply:
corrected